package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "pnpm run '/^lint:.*/'",
     "lint:eslint": "eslint --cache --cache-location 'node_modules/.cache/eslint/.eslintcache' .",
     "lint:prettier": "prettier --cache --list-different .",
-    "lint:tsc": "tsc",
+    "lint:tsc": "tsc && pnpm --filter 'sku' lint:tsc",
```

Comment:
We could run these in parallel if instead there was a separate `lint:tsc:sku` command. The `lint` command above runs all `lint:*` scripts in parallel.